
abuild/postcheck() complains about not putting files in /usr/local

---
 bindings/lua/compat-5.1r5/compat-5.1.lua | 4 ++--
 configure.ac                             | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/bindings/lua/compat-5.1r5/compat-5.1.lua b/bindings/lua/compat-5.1r5/compat-5.1.lua
index b662270..0179122 100644
--- a/bindings/lua/compat-5.1r5/compat-5.1.lua
+++ b/bindings/lua/compat-5.1r5/compat-5.1.lua
@@ -52,8 +52,8 @@ package.path = LUA_PATH or os.getenv("LUA_PATH") or
 package.cpath = LUA_CPATH or os.getenv("LUA_CPATH") or
              "./?.so;" ..
              "./l?.so;" ..
-             "/usr/local/lib/lua/5.0/?.so;" ..
-             "/usr/local/lib/lua/5.0/l?.so"
+             "/usr/lib/lua/5.0/?.so;" ..
+             "/usr/lib/lua/5.0/l?.so"
 
 --
 -- make sure require works with standard libraries
--- a/configure.ac
+++ b/configure.ac
@@ -843,7 +843,7 @@
           AC_MSG_WARN(Setting Lua include and lib flags to defaults in compat-5.1 and lua 5.1 sources)
           LUA_CFLAGS="-I/usr/local/include -I/usr/local/include/lua -I/usr/local/include/lua/$lua_vdot"
           LUA_LFLAGS="-L/usr/local/lib -L/usr/local/lib/lua -L/usr/local/lib/lua/$lua_vdot $lua_libs"
-          LUA_INSTALL_CMOD="/usr/local/lib/lua/$lua_vdot"
+          LUA_INSTALL_CMOD="/usr/lib/lua/$lua_vdot"
         fi
 
         dnl pass additional lua options
